test/e2e/no_user_identity_test.go

```diff
@@ -84,7 +84,7 @@ func TestCreationOfUserAndIdentityIsSkipped(t *testing.T) {
 	t.Run("user and identity stay there when user is deactivated", func(t *testing.T) {
 		// when
 		userSignup, err := wait.For(t, hostAwait.Awaitility, &toolchainv1alpha1.UserSignup{}).
-			Update(signup.Name,
+			Update(signup.Name, hostAwait.Namespace,
```

Comment:
seeing all these changes, I'm wondering if we really need to provide the namespace explicitly? couldn't it be read from the provided "awaitility" instance?
Just to be clear, it's fine to keep it as it is for now and address it later if we figure out that it would simplify the code.

Reply:
There were some update functions like `UpdateServiceAccount`, `UpdateSpaceRequest`,  and `UpdateConfigMap` that had the namespace as input, that's why I changed a little the generic update function

Reply:
OK, I see :+1: Thanks for clarification